### ISSUE TITLE
C# error when inflating enum with a null parameter

### ISSUE THREAD

**Issue by joemukai** [proletariatgames/hxinflate#1]
```
import serialization.*;

enum TestEnum {
  Test(s:String);
}

class Test {
  static function main() {
    var test = serialization.Deflater.run(Test(null));
    serialization.Inflater.run(test);
  }
}
```

Running this targeting cs produces the following error:

```
Unhandled Exception:
Failed to create enum TestEnum.Test(null)
[ERROR] FATAL UNHANDLED EXCEPTION: Failed to create enum TestEnum.Test(null)
```

**Comment by joemukai** [comment]
I'm going to close this issue, it doesn't see to be an issue with hxInflate. Seems like an issue with createEnum when targeting cs in Haxe 3.1.0. `Type.createEnum(TestEnum,'Test', [null]);` also throws an exception

**Comment by lpetre** [comment]
@joemukai is it fixed in haxe 3.2?
